src/components/popover/popover.md

```diff
@@ -0,0 +1,42 @@
+The `popover` component provides functionality somewhere between a modal and a toggletip. It doesn't interrupt flow like a modal would, but provides more than the purely informational function of a toggletip. It is a way to allow for greater branching functionality while staying in-flow.
```

Comment:
We will want to reference the component as `Popover` since that is how it is used. Please update the rest of the references in this file.